packages/story-content-format/src/index.ts

```diff
@@ -343,3 +354,7 @@ function validateRecursiveListNode(value: any, type?: ListNode.Type): RecursiveL
         ? Core.ListNode.validateListNode(value, type, validateBlock)
         : Core.ListNode.validateListNode(value, validateBlock);
 }
+
+export import TableNode = Core.TableNode;
+export import TableRowNode = Core.TableRowNode;
+export import TableCellNode = Core.TableCellNode;
```

Comment:
IMO `Core.TableNode` (along with Row and Cell nodes) should become generic. Otherwise, there's no information about which types of children are allowed inside cells.

This package was supposed to provide very precise typing information about the document structure. 

Look at ParagraphNode, for example:
 https://github.com/prezly/content-format-js/blob/99f291b2712cd0b5f5616633a20ea1d0e7658ecb/packages/story-content-format/src/index.ts#L264